### PR TITLE
GRR Integration - Added valid CSRFToken retrieval function

### PR DESCRIPTION
**Pull request by secdevopsai**
The getCSRFToken function enables 'POST' requests to send valid CSRF Token. grr-set-flows now properly adds valid CSRF token to header

i.e)
`!grr-set-flows client_id=C.120694fab000fff flow="{\"runner_args\":{\"flow_name\":\"Netstat\",\"output_plugins\":[]},\"args\":{}}"`

### CODE REVIEW

File: Integrations/integration-GRR.yml [hunk 1]
```diff
@@ -34,8 +34,21 @@ script:
...
+        var res = http(
+                url,
+                {
+                    Method: 'GET',
+                    Username: username,
+                    Password: password
+                },
+                insecure
+            );
+        return res.Cookies[0].Value;
```
Comment (dorsha) [inline Integrations/integration-GRR.yml]: Just add verification that the request was succeeded 
```
if (res.StatusCode < 200 || res.StatusCode >= 300) {
    throw 'Failed to get CSRF token, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
}
```

Comment (dorsha) [inline Integrations/integration-GRR.yml]: Is the CSRF token will always be the first in the cookies array? I would look for the correct key and then take the value (to be on the safe side).

Comment (secdevopsai) [reply Integrations/integration-GRR.yml]: Complete

Comment (secdevopsai) [reply Integrations/integration-GRR.yml]: Complete

**Comment by secdevopsai** [comment]
@dorsha changes completed. Let me know if there's anything else..

File: Integrations/integration-VMRay.yml [hunk 2]
```diff
@@ -74,8 +74,7 @@ script:
         # and stores it internaly. To get the filename you need an automation.
         # You can use demisto.args()[argName] to get a specific arg. args are strings.
         file = demisto.args()['file_id']
-        path = demisto.getFilePath(file)
```
Comment (orius123) [inline Integrations/integration-VMRay.yml]: I think this a bad merge that overrides a fix. @meirwah right?

Comment (meirwah) [reply Integrations/integration-VMRay.yml]: true, @Spivaka plz fix this merge

**Comment by Spivaka** [comment]
fixes https://github.com/demisto/etc/issues/10038

File: Integrations/integration-VMRay.yml (hunk 2 above)
Comment (secdevopsai) [reply Integrations/integration-VMRay.yml]: accidental, fixing..

**Pull request closed without merging**